Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -6347,29 +6359,27 @@ void IVisualTestingDriver.ChangeAppWindowSize(int width, int height)
 
         }
 
-        void IWindowExplorer.UpdateElementInfoFields(ElementInfo ei)
+        void IWindowExplorer.UpdateElementInfoFields(ElementInfo EI)
         {
```

Comment:
function parameters should be with lower case, instaed "EI" it better be "elementInfo"